strider/query_planner.py

```diff
@@ -358,23 +364,29 @@ async def generate_plan(
                     We couldn't find a KP for every edge in your query graph
                 """
         })
-        raise NoAnswersError("Cannot traverse query graph using KPs")
+        return []
 
-    plan = defaultdict(list)
+    plans = []
 
     for current_qg in filtered_qg_list:
         # Starting at each pinned node, construct a plan
         for pinned in pinned_nodes:
             path_nodes, path_edges = dfs(current_qg, pinned)
-            # If we don't traverse every node we can't use this
-            if set(path_nodes) != set(current_qg['nodes'].keys()):
+            all_nodes = current_qg['nodes'].keys()
+            bound_nodes = [
```

Comment:
Is `bound_nodes == pinned_nodes`?

Reply:
Yes it is. I'll switch to pinned_nodes for consistency in language.